MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,15 +2,164 @@ import UIKit
+    }
+    struct QuizResultsViewModel {
+      let title: String
+      let text: String
+      let buttonText: String
+    }
+    struct QuizQuestion {
+      let image: String
+      let text: String
+      let correctAnswer: Bool
+    }
+
+    private let questions: [QuizQuestion] = [
+    QuizQuestion(image: "The Godfather", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizQuestion(image: "The Dark Knight", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizQuestion(image: "Kill Bill", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizQuestion(image: "The Avengers", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizQuestion(image: "Deadpool", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizQuestion(image: "The Green Knight", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizQuestion(image: "Old", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+    QuizQuestion(image: "The Ice Age Adventures of Buck Wild", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+    QuizQuestion(image: "Tesla", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+    QuizQuestion(image: "Vivarium", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false)
+    ]
     override func viewDidLoad() {
         super.viewDidLoad()
+        let viewModel = convert(model: currentQuestion)
+        show(quiz: viewModel)
+        yesButton.transform = CGAffineTransform(scaleX: 0.9, y: 0.9)
+        yesButton.addTarget(self, action: #selector(yesButtonPressed), for: .touchDown)
+        yesButton.addTarget(self, action: #selector(yesButtonReleased), for: [.touchUpInside, .touchUpOutside])
+        noButton.transform = CGAffineTransform(scaleX: 0.9, y: 0.9)
+        noButton.addTarget(self, action: #selector(noButtonPressed), for: .touchDown)
+        noButton.addTarget(self, action: #selector(noButtonReleased), for: [.touchUpInside, .touchUpOutside])
+    }
+    
+    @IBAction private func yesButtonClicked(_ sender: UIButton) {
+        let currentQuestion = questions[currentQuestionIndex]
+        let givenAnswer = true
+        showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+        
+    }
+    
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        let currentQuestion = questions[currentQuestionIndex]
+        let givenAnswer = false
+        showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+    }
+    
+    private func show(quiz result: QuizResultsViewModel) {
+        let alert = UIAlertController( title: result.title,
+                                       message: result.text,
+                                       preferredStyle: .alert)
+        let action = UIAlertAction(title: result.buttonText, style: .default, handler: { _ in self.currentQuestionIndex = 0
```

Comment:
здесь принято начинать с новой строки
```suggestion
        let action = UIAlertAction(title: result.buttonText, style: .default, handler: { _ in 
            self.currentQuestionIndex = 0
```

Reply:
Исправлено!